app/components/layout/Navigation/index.js

```diff
@@ -21,7 +21,8 @@ class Navigation extends React.Component {
   static propTypes = {
     organization: React.PropTypes.object,
     viewer: React.PropTypes.object,
-    relay: React.PropTypes.object.isRequired
+    relay: React.PropTypes.object.isRequired,
+    assumed: React.PropTypes.bool
```

Comment:
Not sure how to pass this in from where the `Relay.RootContainer` is instantiated